test/iron-overlay-behavior.html

```diff
@@ -1154,11 +1154,26 @@
       test('newest overlay appear on top', function(done) {
         runAfterOpen(overlay1, function() {
           runAfterOpen(overlay2, function() {
-            var styleZ = parseInt(window.getComputedStyle(overlay1).zIndex, 10);
-            var style1Z = parseInt(window.getComputedStyle(overlay2).zIndex, 10);
+            var style1Z = parseInt(window.getComputedStyle(overlay1).zIndex, 10);
+            var style2Z = parseInt(window.getComputedStyle(overlay2).zIndex, 10);
+            var bgStyleZ = parseInt(window.getComputedStyle(overlay1.backdropElement).zIndex, 10);
+            assert.isAbove(style2Z, style1Z, 'overlay1 above overlay2');
```

Comment:
aren't all of these messages backwards? doesn't this mean overlay2 has a higher z index than overlay1 thus 2  is above 1?

Reply:
The way I was considering it is: the message should reflect the wrong state rather than the correct state, as in "Error! overlay1 above overlay2!". Lol, lemmy be positive & write this message to reflect the right state, as in "All good, overlay2 is above overlay1" :p

Reply:
@e111077 done in https://github.com/PolymerElements/iron-overlay-behavior/pull/252/commits/5050abeeb1a08160e44c80b4b64622f5482936df